app/retail/templates/settings/account.html

```diff
@@ -343,7 +343,7 @@ <h3 class="modal-title" id="export-modal-label">{% trans "Download in progress"
 
       $(`#${this.id}`).addClass('bound')
     });
-    $("#close-modal-button").on('click', function() {
+    $("#closeModalButton").on('click', function() {
```

Comment:
I think there is a much better approach we can take here. This was submitting a form before, which we should now bind to the form submit instead of to this button. Then we should submit a request via AJAX and show the modal. There isn't a need to `unbind` the event and then trigger the event.

Reply:
Let's pair on this next week